issue_template.md

```diff
@@ -0,0 +1,19 @@
+## Issue description
+
+If you found a bug or want to propose a feature, please describe it as accurately as possible. Use the following checklists. This way we can help you the best. 
+
+### Your checklist for a bug report
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the issue. If you're unsure about any of them, don't hesitate to ask. We're here to help!_
+
+- [ ] Have you described how to reproduce your bug?
+- [ ] Which operating system, browser and versions do you use?
+- [ ] Can you attach some logs or screenshots to this issue?
+- [ ] Have you already tried solve it yourself? Can you provide any findings?
```

Comment:
This is an important point that should move upwards in the first paragraph. Something like

_If you found a bug or want to have a new feature implemented, please take your time and look whether this issue has been discussed in another issue or our [mailing list](https://groups.google.com/forum/#!forum/scmmanager) already._

_SCM-Manager is open source. If you have any level of programming skills, maybe you can help yourself. We would love to support you._